### PR TITLE
[Sage-216] Radio, Checkbox - Foundations Update

### PR DESCRIPTION
**Pull request by anechol**
## Description

Updates Radio and Checkbox components:
- Adjusts left margin of label to `12px`
- Adjusts message margin to match label indent
- Updates input size to `20px`
- Updates components with correct Sage colors
- Increases focus outline size to `4px`

### Radio
- Adds custom content slot

[Figma](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/%E2%9A%A0%EF%B8%8F-%5BDO-NOT-USE-WIP%5D-Sage-Update?node-id=557%3A28030)

## Screenshots

|  Before  |  After  |
|--------|--------|
Screen Shot 2022-02-17 at 4 14 45 PM|Screen Shot 2022-02-17 at 4 13 21 PM
Screen Shot 2022-02-17 at 4 14 36 PM|Screen Shot 2022-02-17 at 4 12 54 PM

### Rails
- View [Checkbox](http://localhost:4000/pages/component/checkbox) and [Radio](http://localhost:4000/pages/component/radio)
- Check that style updates are present with correct colors.
- Check that custom content slot for Radio works as intended

### React
- View [Checkbox](http://localhost:4100/?path=/docs/sage-checkbox--default) and [Radio](http://localhost:4100/?path=/docs/sage-radio--default)
- Check that style updates are present with correct colors.

## Testing in `kajabi-products`

(**LOW**) Updates Radio and Checkbox component for Foundations.

## Related

[Sage-216](https://kajabi.atlassian.net/browse/SAGE-216)

### CODE REVIEW

**Review by philschanely — APPROVED** [review]
Looking good here! 🙌

File: docs/app/views/examples/components/checkbox/_props.html.erb [hunk 1]
```diff
@@ -70,3 +70,14 @@
...
+  <td colspan="4"><strong>Content Slots</strong></td>
+</tr>
+<tr>
+  <td><%= md('`:sage_checkbox_custom_content`') %></td>
+  <td><%= md('
+  Slot for any custom content below the checkbox.
+  ') %></td>
+  <td></td>
+  <td></td>
+</tr>
```
Comment (pixelflips) [inline docs/app/views/examples/components/checkbox/_props.html.erb:83]: Nitpick: missing newline

File: docs/app/views/examples/components/radio/_props.html.erb [hunk 2]
```diff
@@ -76,3 +76,14 @@
...
+  <td colspan="4"><strong>Content Slots</strong></td>
+</tr>
+<tr>
+  <td><%= md('`:sage_radio_custom_content`') %></td>
+  <td><%= md('
+  Slot for any custom content below the radio.
+  ') %></td>
+  <td></td>
+  <td></td>
+</tr>
```
Comment (pixelflips) [inline docs/app/views/examples/components/radio/_props.html.erb:89]: Nitpick: missing newline

**Review by pixelflips — CHANGES_REQUESTED** [review]
This looks great. A couple of tiny nitpicks and one small thing I noticed that may not be related to these updates. 

The disabled/checked radio gets an outline applied on hover. The disabled/checked checkbox does not. Not sure which is actually correct.

Screen Shot 2022-02-17 at 2 50 30 PM

**Review by pixelflips — APPROVED** [review]
👍🏼  LGTM! Great work. Thanks for resolving those minor issues!

**Pull request merged**